Automattic/jetpack
# Jetpack: redirect to the Jetpack page on multisites after activation

Related to #35449
Fixes #35557

## Proposed changes:
When Jetpack is activated, we redirect to My Jetpack, which is unavailable on multisites.
This PR will redirect users on multisites to the Jetpack page.

### Other information:


## Jetpack product discussion
p1707442008656029-slack-CA17PSW87

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
1. Setup a non-multisite site.
2. Go to "Plugins" and deactivate Jetpack and all standalone plugins.
3. Activate Jetpack. Confirm you got redirected to the "My Jetpack" page.
4. If you're using a new site without any standalone plugins installed, "My Jetpack" should display the "Welcome to Jetpack!" banner.
5. Setup a multisite, go to "Plugins" and deactivate Jetpack and all standalone plugins.
6. Network-activate Jetpack, confirm you're redirected to `/network/admin.php?page=jetpack`.
7. Network-deactivate Jetpack.
8. Activate Jetpack on a subsite, confirm you're redirected to the Jetpack's connect screen.
